cmd/root.go

```diff
@@ -172,6 +172,17 @@ Instance Level Configuration
     	    my-project:us-central1:my-db-server \
     	    'my-project:us-central1:my-other-server?address=0.0.0.0&port=7000'
 
+Instance Level Configuration Parameters:
```

Comment:
Let's leave this section off and immediately go into the particular parameter here. For example,

When necessary, you may specify the full path to a Unix socket using the `unix-socket-path` query parameter. The value of the parameter should be an absolute path to where you want the socket to be created. Parent directories must exist, or the socket creation will fail. For Postgres instances, the proxy will ensure that the last path element is '.s.PGSQL.5432', appending it if necessary. For example,

./cloud-sql-proxy \
    	    'my-project:us-central1:my-db-server?unix-socket-path=/path/to/socket'

Reply:
Updated with a few minor edits.